features/other_b/skin.feature

```diff
@@ -67,6 +73,33 @@ Feature: creating and editing skins
   When I am on skin's new page
     Then I should see "Sorry, you don't have permission"
 
+  Scenario: An admin can reject a skin
+  Given the unapproved public skin "public skin"
+    And I am logged in as an admin
+  When I go to admin's skins page
+    And I check "make_rejected_public_skin"
```

Comment:
(There's no label on the checkboxes for rejecting and unrejecting, so this is fine. ;))